includes/Classifai/Admin/templates/onboarding-step-four.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * Step-4 template for ClassifAI Onboarding.
+ *
+ * @package ClassifAI
+ */
+
+$onboarding_options   = get_option( 'classifai_onboarding_options', array() );
+$enabled_features     = $onboarding_options['enabled_features'] ?? array();
+$configured_providers = $onboarding_options['configured_providers'] ?? array();
+$onboarding           = new Classifai\Admin\Onboarding();
+$features             = array_filter(
+	$onboarding->get_features(),
+	function( $feature ) use ( $enabled_features, $configured_providers ) {
+		return ! empty( $feature['features'] ) && array_intersect( array_keys( $feature['features'] ), array_keys( $enabled_features ) ) && array_intersect( array_keys( $feature['features'] ), $configured_providers );
+	}
+);
+
+$args = array(
+	'step'       => 4,
+	'title'      => __( 'Welcome to ClassifAI', 'classifai' ),
+	'image'      => 'https://via.placeholder.com/334x334',
+	'left_link'  => array(
+		'text' => __( 'Adjust ClassifAI settings', 'classifai' ),
+		'url'  => admin_url( 'admin.php?page=classifai_settings' ),
+	),
+	'right_link' => array(
+		'text'   => __( 'Done', 'classifai' ),
+		'submit' => false,
+		'url'    => admin_url(),
+	),
+);
+
+// Header
+require_once 'onboarding-header.php';
+?>
+
+<h1 class="classifai-setup-title">
```

Comment:
Since h1 is already exist, lets change it to some other tag like h2

Reply:
Nice catch!